images/linux/scripts/installers/docker.sh

```diff
@@ -1,25 +1,27 @@
 #!/bin/bash -e
 ################################################################################
-##  File:  docker-moby.sh
+##  File:  docker.sh
 ##  Desc:  Installs docker onto the image
 ################################################################################
 
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/install.sh
 
-# Check to see if docker is already installed
-docker_package=moby
-echo "Determing if Docker ($docker_package) is installed"
-if ! IsPackageInstalled $docker_package; then
-    echo "Docker ($docker_package) was not found. Installing..."
-    apt-get remove -y moby-engine moby-cli
-    apt-get update
-    apt-get install -y moby-engine moby-cli
-    apt-get install --no-install-recommends -y moby-buildx
-    apt-get install -y moby-compose
-else
-    echo "Docker ($docker_package) is already installed"
-fi
+repo_url="https://download.docker.com/linux/ubuntu"
+gpg_key="/usr/share/keyrings/docker.gpg"
+repo_path="/etc/apt/sources.list.d/docker.list"
+
+apt-get update
+apt-get install ca-certificates curl gnupg2
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o $gpg_key
```

Comment:
we can probably use the `os.sh` helper and use the `getOSVersionLabel` function to get a code name